hack/update-codegen.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
Any reason not to put this script in the `bin/` dir? Putting it in `hack/` makes it seem.... hacky.